migrations/emailmonitor/oxa_email_notifier.py

```diff
@@ -0,0 +1,239 @@
+
+    return msg
+
+def MarkDatabaseActivationEMailSent( row ):
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    dt = datetime.now()
+    sql = "UPDATE oxa.oxa_activationfailed SET is_processed = 1, date_processed = '"+str(dt)+"' WHERE id="+str(row[0])
+    cursor.execute(sql)
+    sql = "UPDATE oxa.oxa_activationsummary SET resend = resend+1 WHERE xyear="+str(row[1])+" and xmonth="+str(row[2])+" and xday="+str(row[3])    
+    cursor.execute(sql)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+email_monitor_receivers_eng = ""
+email_monitor_receivers_sup = ""
+
+WriteLog("Started running process")
+dt = datetime.now()
+subject = dt.strftime("%Y-%m-%d %H:%M:%S")
+filename = dt.strftime("%Y%m%d%H%M%S") + ".mail.txt"
+f = open("/oxa/"+filename,"w")
+
+OutputMonthlySummary(f)
+OutputCurrentMonthSummary(f)
+OutputAllFailedEmailActivation(f)
+f.close()
+
+mailserver = None
+try:
+    mailserver = GetMailServerConnection()
+except Exception, ex:
+    WriteLog("Email Server Connection Error",ex)
+
+if mailserver != None:
+    try:
+        results = GetNotProcessedEmailActivationFailures() 
+        activated_emails = "Activation email sent to these users:\r\n\r\n"
+        activated_emails_keys = "Activation email sent to these users:\r\n\r\n"
+        activated_count = 0
+        for row in results:
+            msg = GetActivationEmailForUser(row[6])
+            try:
+                SendMail(mailserver,msg,row[5],"Activate Your Microsoft Learning Account")
+                MarkDatabaseActivationEMailSent(row)
+                WriteLog("Activation email is sent to user "+row[5]+" with key "+row[6])
+                activated_count = activated_count + 1
+                activated_emails = activated_emails + row[5]+"\r\n"
+                activated_emails_keys = activated_emails_keys + '{:50s} {:36s}'.format(row[5],row[6])+"\r\n"
+
+            except Exception, ex:
+                WriteLog("Failed to send activation key to user "+row[5] + " with key " + row[6],ex) 
+        
+        dt = datetime.now()
+        subject = dt.strftime("%Y-%m-%d %H:%M:%S")
+        filename = dt.strftime("%Y%m%d%H%M%S") + ".mail.txt"
+        f = open("/oxa/"+filename,"w")
```

Comment:
nit: please surround operators with spaces. there are also a few places where commas should be followed by a space. There are linter plugins for VS Code that I'm a big fan of too https://www.google.com/?gws_rd=ssl#q=linter+python+vs+code&spf=1497372265134